data/job-board-rules-guidelines.md

```diff
@@ -0,0 +1,28 @@
+### 1. Job Submission and Approval Process:
+
+- After successful job submission, each job will undergo a verification process by the PyCon India 2023 team.
+- The approval process may take approximately 4-5 business days.
+- Approved jobs will be made visible on the job board once they pass the verification process.
+- Submissions that do not meet the guidelines may be rejected, and submitters will be notified of the reasons for rejection.
+
+### 2. Timestamp Order:
+
+- Job posts are sorted based on the timestamp they are submitted. The first job submitted will be visible first, followed by subsequent submissions in chronological order.
```

Comment:
Submitted or selected/approved by the team? 
Which one of them will be shown first?

Reply:
Once the submission done google form, we (internal team) will be able to review the posting and if it looks good (i.e no spam) it can be marked for approval and will automatically gets pubslished on the website.

Sorting of postings is done based on submission time/date.

Previous live example here: https://in.pycon.org/2021/jobs/